pools/boosts/index.json

```
@@ -0,0 +1,14 @@
+[
+    {
+        "id": "boosts_aave",
```

I'd maybe rename, just a suggestion:
```suggestion
        "id": "boosted_aave",
```